src/goto-programs/compute_called_functions.cpp

```diff
@@ -91,11 +90,10 @@ std::set<irep_idt> compute_called_functions(
     irep_idt id=*working_queue.begin();
     working_queue.erase(working_queue.begin());
 
-    if(done.find(id)!=done.end())
+    if(functions.find(id)!=functions.end())
```

Comment:
A few nit picks (placing them here for github doesn't feature commenting on commit messages):
1) Right here: clang-format preferred spacing: add spaces around `!=`.
2) You might actually do:
```
if(!functions.insert(id).second)
  continue;
```
and thereby further improve performance by not duplicating the log-time search of the separate find+insert steps.
2) Commit message 1: Please add "Fixes: #1783" to auto-close the issue.
3) Commit message 2: Please spell out "Function-pointer" (instead of fp) to avoid confusion, e.g., with "floating-point."